Force install path of tcltk lib conforming to the FHS
--- a/configure.ac
+++ b/configure.ac
@@ -815,7 +815,7 @@
 		# might contain staging_dir prefix when cross-compiling.
 		OWTCL_INSTALL_PATH="`echo ${TCL_PACKAGE_PATH} | cut -d' ' -f1`"
 		# Debian Hack: do not install in /usr/local/lib/tcltk
-		# OWTCL_INSTALL_PATH="/usr/lib/tcltk"
+		OWTCL_INSTALL_PATH="/usr/lib/tcltk"
 		AC_SUBST(OWTCL_INSTALL_PATH)
 	fi
 fi
